app/src/androidTest/java/org/mozilla/focus/activity/ExitAppTest.kt

```diff
@@ -0,0 +1,84 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.activity
+
+import android.Manifest
+import android.content.Intent
+import android.support.annotation.Keep
+import android.support.test.rule.ActivityTestRule
+import android.support.test.rule.GrantPermissionRule
+import android.support.test.runner.AndroidJUnit4
+import junit.framework.TestCase.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.focus.autobot.session
+import org.mozilla.focus.utils.AndroidTestUtils
+
+@Keep
+@RunWith(AndroidJUnit4::class)
+class ExitAppTest {
+
+    @JvmField
```

Comment:
You don't need @JvmField unless someone from the Java world will call it

Reply:
remove grant permission rule